src/pae/ieee802_1x_kay.h

```diff
@@ -114,24 +117,26 @@ struct receive_sc {
 
 	struct os_time created_time; /* Time createdTime */
 
+	u32 ssci; /* SSCI - XPN cipher suites only */
+
 	struct dl_list list;
 	struct dl_list sa_list;
 };
 
 /* ReceiveSA in IEEE Std 802.1AE-2006, Figure 10-6 */
 struct receive_sa {
```

Comment:
I am not sure why you want to re-order the structure here, but it makes code review harder. can you keep the existing order and just add fields you need?

Reply:
The receive_sc structure was re-ordered to match the ordering of the similar transmit_sc structure for improved consistency.